server/container_list.go

```diff
@@ -79,7 +79,7 @@ func (s *Server) ListContainers(ctx context.Context, req *pb.ListContainersReque
 			continue
 		}
 		podSandboxID := ctr.Sandbox()
-		cState := ctr.StateNoLock()
+		cState := ctr.State()
```

Comment:
Do we need these changes to State()? I don't think we do.

Reply:
Addressed